From 5ff36963e581087c16e6f3f6dcdd13aa65a48fbe Mon Sep 17 00:00:00 2001
From: Pierre K <pierrekn@gmail.com>
Date: Wed, 17 Aug 2022 17:51:56 +0200
Subject: [PATCH 173/196] Fix building on ubuntu 22.04 (#262)

Adds missing <optional> header. This causes a build failure on
GCC 11.
---
 src/script/interpreter.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/script/interpreter.h b/src/script/interpreter.h
index 3407252d8..c115875b1 100644
--- a/src/script/interpreter.h
+++ b/src/script/interpreter.h
@@ -15,6 +15,7 @@
 #include <script/sighashtype.h>
 
 #include <cstdint>
+#include <optional>
 #include <vector>
 
 class CPubKey;
-- 
2.39.5 (Apple Git-154)

